src/EnergyPlus/EconomicLifeCycleCost.cc

```diff
@@ -1956,19 +1935,19 @@ void WriteTabularLifeCycleCostReport(EnergyPlusData &state)
         columnWidth.dimension(numColumns, 14); // array assignment - same for all columns
         tableBody.allocate(numColumns, elcc->lengthStudyYears);
         tableBody = "";
-        for (iYear = 1; iYear <= elcc->lengthStudyYears; ++iYear) {
+        for (int iYear = 1; iYear <= elcc->lengthStudyYears; ++iYear) {
             rowHead(iYear) = format("{} {}", UtilityRoutines::MonthNamesCC[static_cast<int>(elcc->baseDateMonth)], elcc->baseDateYear + iYear - 1);
         }
-        for (jObj = 0; jObj < elcc->numResourcesUsed; ++jObj) {
-            curCashFlow = CostCategory::Num + elcc->numRecurringCosts + elcc->numNonrecurringCost + jObj;
+        for (int jObj = 0; jObj < elcc->numResourcesUsed; ++jObj) {
+            int curCashFlow = CostCategory::Num + elcc->numRecurringCosts + elcc->numNonrecurringCost + jObj;
             columnHead(jObj + 1) = elcc->CashFlow[curCashFlow].name;
-            for (iYear = 1; iYear <= elcc->lengthStudyYears; ++iYear) {
+            for (int iYear = 1; iYear <= elcc->lengthStudyYears; ++iYear) {
                 tableBody(jObj + 1, iYear) = RealToStr(elcc->CashFlow[curCashFlow].yrAmount(iYear), 2);
             }
         }
         columnHead(numColumns) = Total;
-        for (iYear = 1; iYear <= elcc->lengthStudyYears; ++iYear) {
-            tableBody(jObj + 1, iYear) =
+        for (int iYear = 1; iYear <= elcc->lengthStudyYears; ++iYear) {
+            tableBody(elcc->numResourcesUsed + 1, iYear) =
                 RealToStr(elcc->CashFlow[CostCategory::TotEnergy].yrAmount(iYear) + elcc->CashFlow[CostCategory::Water].yrAmount(iYear), 2);
```

Comment:
Code change. jObj (old line 1971) is a for loop counter at line 1941 (old line 1962). That loop should end with jObj = elcc->numResourcesUsed. The change at line 1950 should be equivalent to using jObj. With issue #9903 I am suspicious of any code change.